scripts/build/Platform/Linux/build_config.json

```diff
@@ -83,7 +83,7 @@
       "CMAKE_OPTIONS": "-G 'Ninja Multi-Config' -DCMAKE_C_COMPILER=clang-12 -DCMAKE_CXX_COMPILER=clang++-12 -DLY_PARALLEL_LINK_JOBS=4",
       "CMAKE_LY_PROJECTS": "AutomatedTesting",
       "CMAKE_TARGET": "all",
-      "CTEST_OPTIONS": "-E Gem::EMotionFX.Editor.Tests -LE (SUITE_sandbox|SUITE_awsi) -L FRAMEWORK_googletest --no-tests=error",
+      "CTEST_OPTIONS": "-E (AutomatedTesting::Atom_TestSuite_Main|AutomatedTesting::PhysicsTests_Main|AutomatedTesting::PrefabTests|Gem::EMotionFX.Editor.Tests) -L (SUITE_smoke|SUITE_main) -LE (REQUIRES_gpu) --no-tests=error",
```

Comment:
Instead of disabling tests here, which is not trivial to find since it's unrelated to the tests themselves, it would be better if each test folder would disable itself, that way each test group would only have to modify their own tests' files inside AutomatedTesting project, instead of this Jenkins build script.

Isn't there a way that each python test suite can disable itself for a platform in its CMakeLists.txt inside AutomatedTesting project?

Reply:
This seemed like a simpler change to make, and I plan on writing an issue to track re-enabling each of these that instructs to re-enable by modifying this file.  It is possible to conditionally skip inside each Python module.

Reply:
I agree with Aaron, disabling tests for specific platform it shouldn't be done here.
Team members shouldn't be modifiying this common file for specifying the support. There are tests that won't be available for Linux in the short term 

This approach creates tech-debt for shortcutting a way to ignore tests on a platform. A mistake in this long filter can cause unexpectedly removing tests by accident.

I strongly suggest to have an alternative way to do this, a simple suggestion could be to add something like UNSUPPORTED_PLATFORMS in ly_add_pytest, like:

    ly_add_pytest(
        NAME AutomatedTesting::PhysicsTests_Main
        TEST_SUITE main
        TEST_SERIAL
        UNSUPPORTED_PLATFORMS LINUX
        PATH ${CMAKE_CURRENT_LIST_DIR}/TestSuite_Main.py
        RUNTIME_DEPENDENCIES
            Legacy::Editor
            AZ::AssetProcessor
            AutomatedTesting.Assets
        COMPONENT
            Physics
    )

Reply:
An easy way to disable independently test suites by platform, like @AMZN-AlexOteiza is suggesting, seems like the right way to go.

My 2 cents is that if this change is a short-term one, owned by @Kadino and he's going to revert it soon, then maybe it's ok to have the tests filtered out in the `build_config.json` file.

But if this is going to be medium/long term change and this is going to pass through multiple teams... then I think to offer a nice way to disable test suite by platforms is the way to go.

Reply:
Another approach: It's only 4 tests groups, it wouldn't be too much work to just do it with PAL files, like Whitebox or Cloth tests are doing to only run on windows.

Reply:
Correct, I am going to help drive other SIGs to fix any issues with their tests. The tests will not be disabled from the perspective of a local machine, and will only avoid running in the two flavors of Jenkins AR jobs.

Reply:
I am mostly fine, the only caveat is that it will be easier for a developer to enable the test locally and be able to run it and then assume it will run in AR.
If we disable it in the `build_config.json`, once they fix the issues, they will have to remember/be aware that they need to enable it in `build_config.json` for AR/Jenkins to run it. 
My concern is a test working locally, them sending it to PR, Jenkins passing (because is disabled) and later on that test not working in Jenkins.

If the test is disabled for that platform at the registration level (in cmake), they have to enable it locally to test it/validate it and thats all they need to do.

I am fine either way, just wanted to warn that the `build_config.json` approach will require to be more cautious during PR review.